docs/user/config-file/v2.rst

```diff
@@ -297,7 +297,7 @@ conda.environment
 The path to the Conda `environment file <https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_, relative to the root of the project.
 
 :Type: ``path``
-:Required: ``true``
+:Required: ``false``
```

Comment:
```suggestion
:Required: when using conda key
```

Reply:
I'll leave this change for later for now, as we have a number of fields we should do this for.